README.md

```diff
@@ -378,27 +398,10 @@ export default Ember.Route.extend({
 });
 ```
 
-# Installation
```

Comment:
I am not sure that removing this section from the README should be done.
Most Ember Addons have this as part of the README and I feel that it is expected of Addons to have this section.

@bendemboski @knownasilya What do you think?

Reply:
Contribution information for Ember Addons is now split by default https://github.com/ember-cli/ember-cli/pull/8203

More context:
https://noti.st/kennethlarsen/jXt2dj/the-state-of-community-documentation#sE2Y4Nq

Reply:
This is a different installation section. This is for the consumer not the contributor.